### PR TITLE
Update functions.lua

### PR DESCRIPTION
**Pull request by SKITTLE6969**
## Description

fix to GetClosestVehicle & GetClosestObject  if a null value is passed

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Comment by MafewTM** [comment]
Pretty sure these weren't added on purpose but we'll see

**Comment by D4isDAVID** [comment]
> Pretty sure these weren't added on purpose but we'll see

Nope. I just forgot/didn't notice that these weren't added. We need this for backwards compatibility.

File: bridge/qb/client/functions.lua [hunk 1]
```diff
@@ -183,13 +183,23 @@ end
 
 ---@deprecated use lib.getClosestVehicle from ox_lib
 functions.GetClosestVehicle = function(coords)
+    if coords then
+        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
+    else
+        coords = GetEntityCoords(cache.ped)
+    end
```
Comment (D4isDAVID) [inline bridge/qb/client/functions.lua:190]:
```suggestion
    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
```
This part can be reduced.

File: bridge/qb/client/functions.lua [hunk 2]
```diff
@@ -183,13 +183,23 @@ end
...
     return closestVehicle, vehicleDistance
 end
 
 ---@deprecated use lib.getClosestObject from ox_lib
 functions.GetClosestObject = function(coords)
+    if coords then
+        coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords
+    else
+        coords = GetEntityCoords(cache.ped)
+    end
```
Comment (D4isDAVID) [inline bridge/qb/client/functions.lua:202]:
```suggestion
    coords = type(coords) == 'table' and vec3(coords.x, coords.y, coords.z) or coords or GetEntityCoords(cache.ped)
```
Same here.

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

**Review by MafewTM — APPROVED** [review]

**Review by D4isDAVID — APPROVED** [review]

**Review by solareon — APPROVED** [review]

**Pull request merged**